gdmix-workflow/test/resources/detext-movieLens.yaml

```diff
@@ -11,30 +11,33 @@
       "num_filters": 50,
       "num_train_steps": 1000,
       "num_units": 64,
-      "optimizer": "bert_adam",
+      "optimizer": "adamw",
       "pmetric": "auc",
-      "all_metrics": ["auc"],
       "steps_per_stats": 10,
       "steps_per_eval": 100,
       "train_batch_size": 64,
       "test_batch_size": 64,
       "vocab_file": "movieLens/detext/vocab.txt",
       "resume_training": false,
-      "train_file": "movieLens/detext/trainingData/train_data.tfrecord",
-      "dev_file": "movieLens/detext/validationData/test_data.tfrecord",
-      "test_file": "movieLens/detext/validationData/test_data.tfrecord",
+      "train_file": "movieLens/detext/trainingData",
```

Comment:
This is not a file, but a folder.

Reply:
Yes, but passing a file will raise an error. The parameter name is from DeText argument(https://github.com/linkedin/detext/blob/master/src/detext/args.py), we can't change it from our end.